src/modules/map/components/LocationPopup.vue

```diff
@@ -159,9 +160,10 @@ function clearClick() {
     <component
         :is="mappingFrameworkSpecificPopup"
         v-if="coordinate"
-        :title="selectedTab == 'position' ? i18n.t('position') : i18n.t('link_bowl_crosshair')"
+        :title="selectedTab === 'position' ? i18n.t('position') : i18n.t('link_bowl_crosshair')"
         :coordinates="coordinate"
         :projection="projection"
+        :mode="MapPopoverMode.FEATURE_TOOLTIP"
```

Comment:
In FEATURE_TOOLTIP mode, when collapsing the feature it change slightly the size of the window and make the center of the tooltip moving, which is not good with the location popup. The later has already a bug when changing to sharing tab which also change it size.

Reply:
I found a way to fix the location popup issue, which was mainly due to network share buttons overflowing. They now adapt to the available space they have, and wrap if needed (CSS Grid instead of CSS Flex)

Reply:
Note that on my PR #837 I set a fix width to solve the location popup arrow moving